tests/macsec/__init__.py

```diff
@@ -46,27 +49,27 @@ def pytest_generate_tests(self, metafunc):
             metafunc.parametrize('macsec_profile',
                                  [self.macsec_profiles[x] for x in profiles],
                                  ids=profiles,
-                                 scope="session")
+                                 scope="module")
 
-    @pytest.fixture(scope="session")
+    @pytest.fixture(scope="module")
```

Comment:
Just curious, does "session" or 'module" make any difference in our pytest framework?
We always run one module in a session, by "-c module" parameter.

Reply:
No difference. But the fixture, sanity_check, is module level. Meanwhile, the session fixtures are always executed before the module fixtures. So, I change macsec fixture to guarantee the MACsec setup will execute after sanity_check.